src/goto-instrument/contracts/assigns.h

```diff
@@ -22,6 +22,27 @@ Date: July 2021
 
 typedef std::pair<const exprt, const exprt> slicet;
 
+/// A guarded slice of memory
```

Comment:
`\brief`? Why struct and not a class here?

Reply:
there are no methods associated with this, it is just structured data